src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -460,6 +464,26 @@ public ApiResponse handleApiView(String name, JSONObject params) throws ApiExcep
 				resultList.addItem(new ApiResponseSet("scan", map));
 			}
 			result = resultList;
+		} else if (VIEW_URLS.equals(name)) {
+			ApiResponseList resultUrls = new ApiResponseList(name);
+			SpiderScan scan = (SpiderScan) this.getSpiderScan(params);
+			synchronized (scan.getResourcesFound()) {
+				for (SpiderResource sr : scan.getResourcesFound()) {
```

Comment:
I guess it could build two sets kind of like fullResults (VIEW_FULL_RESULTS) does...